index.html

```diff
@@ -0,0 +1,281 @@
+            </div>
+
+            <!--Card Five-->
+            <div class="card flg">
+                <div class="card-text">
+                    <h2 class="project-title">
+                        Multi-Post Stories Gain+Glory
+                    </h2>
+                    <ul class="tags1 flg">
+                        <li class="inner-tag1 flg"><h5 class="flg">Ruby on rails</h5></li>
+                        <li class="inner-tag1 flg"><h5 class="flg">css</h5></li>
+                        <li class="inner-tag1 flg"><h5 class="flg">JavaScript</h5></li>
+                        <li class="inner-tag1 flg"><h5 class="flg">html</h5></li>
+                    </ul>
+                    <div class="see-project">
+                        <button class="btn" type="button">
+                            <span class="btn-text flg">See Project</span>
+                        </button>
+                    </div>
+                </div>
+                <!--Image placeholder goes here-->
+                <div class="img-holder"></div>
+            </div>
+
+            <!--Card Six-->
+            <div class="card flg">
+                <div class="card-text">
+                    <h2 class="project-title">
+                        Multi-Post Stories Gain+Glory
+                    </h2>
+                    <ul class="tags1 flg">
+                        <li class="inner-tag1 flg"><h5 class="flg">Ruby on rails</h5></li>
+                        <li class="inner-tag1 flg"><h5 class="flg">css</h5></li>
+                        <li class="inner-tag1 flg"><h5 class="flg">JavaScript</h5></li>
+                        <li class="inner-tag1 flg"><h5 class="flg">html</h5></li>
+                    </ul>
+                    <div class="see-project">
+                        <button class="btn" type="button">
+                            <span class="btn-text flg">See Project</span>
+                        </button>
+                    </div>
+                </div>
+                <!--Image placeholder goes here-->
+                <div class="img-holder"></div>
+            </div>
+        </div>
+    </section>
+
+    <!-- About me section-->
+    <section id="about-me">
+        <!-- About me text -->
+        <div id="about-me-text" class="flg">
+            <h1 class="primary flg">About me</h1>
+            <p class="support flg">
+                I can help you build a product , feature or website Look through some of 
+                my work and experience! If you like what you see and have a 
+                project you need coded, don’t hestiate to contact me.
+            </p>
+            <button class="btn-resume" type="button">
+                <span class="btn-text flg">Get My Resume</span>
+            </button>    
+        </div>
+
+        <!--About me bacground images-->
+        <div id="background-image2"><!--Image inserted through css--></div>
+        <div id="background-image3"><!--Image inserted through css--></div>
```

Comment:
It's not the best idea to use two images as a background image in one container. the images could easily get disoriented. please check the Figma again and export one background image containing the two SVG designs on it.